src/signals/incident-management/containers/IncidentSplitContainer/components/IncidentSplitFormIncident/__tests__/IncidentSplitFormIncident.test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+
+import { withAppContext } from 'test/utils';
+
+import { subcategoriesWithUniqueKeys as subcategories } from 'utils/__tests__/fixtures';
+import parentIncidentFixture from '../../../__tests__/parentIncidentFixture.json';
+
+import IncidentSplitFormIncident from '..';
+
+describe('IncidentSplitFormIncident', () => {
+  const register = jest.fn();
+  const props = { parentIncident: parentIncidentFixture, subcategories, register };
+
+  it('renders one splitted incident by default', () => {
```

Comment:
Past tense of *split* is *split*, not *splitted*.

Reply:
@boris-arkenaar  AFAIK: `splitted` means `gesplitst' please correct me if this is wrong in this context.